web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1687,7 +1687,7 @@ export const formatters: Formatters = {
       affected_resource_name,
       affected_resource_source,
     }) =>
-      `${affected_resource_kind || 'Node'} [${affected_resource_name}/${affected_resource_source}] changed an access path`,
+      `Access path for ${affected_resource_kind || 'Node'} [${affected_resource_name}/${affected_resource_source}] changed`,
```

Comment:
FYI @tigrato its worth noting that it seems TAG never sets this, so we always display Node.

Reply:
I fixed this bug this morning. Thanks for pointing it out.